### PR TITLE
change the return string of str(Eq) and str(Ne)

### PR DESCRIPTION
**Pull request by luvagarwal**
#8920

### CODE REVIEW

**Comment by smichr** [comment]
When you correct the errors, please also improve the commit message (see Writing Commit Message [at the workflow-wiki](https://github.com/sympy/sympy/wiki/Development-workflow). Also include the text "fixes #8920" in the description/opening paragraph of this PR. I see that you included the reference number in the commit message and in the PR title but there are two problems:

1. it doesn't follow the [syntax for automatically closing](https://help.github.com/articles/closing-issues-via-commit-messages/) the issue (:two: features are right, but the additional "the" broke the syntax)
1. it doesn't provide a link that one can use when viewing this PR

File: sympy/printing/str.py [hunk 1]
```diff
@@ -554,6 +554,15 @@ def _print_Float(self, expr):
         return rv
 
     def _print_Relational(self, expr):
+
+        charmap = {
+            "==": "Eq",
+            "!=": "Ne",
+        }
+
+        if expr.rel_op == '==' or expr.rel_op == '!=':
```
Comment (smichr) [inline sympy/printing/str.py]: how about `if expr.rel_op in charmap:` ?

**Comment by smichr** [comment]
Thanks for working on this! This is going to be a nice change for anyone trying to copy and use the results containing Piecewise with these sorts of things appearing in the conditions...I had to learn about this the hard way recently :-)

**Comment by kaushik94** [comment]
@luviiit  you have to fix all the occurrences of `Eq` appearing as `==` I guess. Or there could be a workaround for this.

**Comment by asmeurer** [comment]
To expand on what @smichr said, please make the first line of your commit message a description of the change itself. The "Fixes #123" should be at the end. Someone should be able to tell what your change does from the commit message alone, without having to look up the issue or read the diff.

**Comment by smichr** [comment]
> The "Fixes #123" should be at the end

preferably not there at all but in the PR description where it is 1) useful for review, also closes the issue (as it does in the commit message), and 3) is clickable. I've updated the wiki.

Also, could you please change the title of the PR? It's not very useful to have an issue number that can't be clicked there, and the number itself doesn't tell the reviewer what they are going to be seeing. (And when there are lots of Issue ### titles it's hard to remember what's what.)

:eyeglasses: See the section **Writing pull request title and description** at [the wiki](https://github.com/sympy/sympy/wiki/Development-workflow)

**Comment by luviiit** [comment]
@kaushik94: Thanks. Changing all the occurences of ```Eq``` appearing as ```==``` manually will take a lot of time.

**Comment by luviiit** [comment]
@smichr, @asmeurer: is there some possible workaround to change all the occurences of ```Eq``` appearing as ```==```. If not, then I will have to see all the tests at Travis that failed, run them locally, which sometimes take a lot of time, and modify them manually.
Thanks :)

**Comment by smichr** [comment]
Does pprint print them with the `==` and `!=`?

**Comment by luviiit** [comment]
@smichr, I have not changed anything in pretty print. So it works the same way as before. It prints ```=``` for ```Eq``` and ```!=``` for ```Ne```.

**Comment by asmeurer** [comment]
Yeah, pprint should stay the same.

**Comment by asmeurer** [comment]
This looks good. Thanks for making this change. We should have done this a long time ago.

**Pull request merged**